modules/apps/object/object-service/src/main/java/com/liferay/object/internal/field/business/type/FormulaObjectFieldBusinessType.java

```diff
@@ -124,9 +124,15 @@ public void validateObjectFieldSettings(
 			objectFieldSetting -> objectFieldSettingsValues.put(
 				objectFieldSetting.getName(), objectFieldSetting.getValue()));
 
+		String script = objectFieldSettingsValues.get("script");
```

Comment:
This commit could be together in the first one when you added this class because this validation is not complex (squash). I think it is easier to review when the work is complete and together :smile:

Reply:
makes sense. thx